packages/default/scss/pivotgrid/_layout.scss

```diff
@@ -1,4 +1,453 @@
+        }
+    }
+
+    .k-pivotgrid-configurator-overlay {
+        position: absolute;
+        overflow: hidden;
+        z-index: 2;
+
+        .k-flex-row & {
+            right: $pivotgrid-configurator-button-size;
+        }
+        .k-flex-row-reverse & {
+            left: $pivotgrid-configurator-button-size;
+        }
+        .k-flex-col & {
+            bottom: $pivotgrid-configurator-button-size;
+        }
+        .k-flex-col-reverse & {
+            top: $pivotgrid-configurator-button-size;
+        }
+    }
+
+
+    // Panel Content
+    .k-pivotgrid-configurator-header {
+        font-size: $pivotgrid-configurator-header-font-size;
+        font-weight: $pivotgrid-configurator-header-font-weight;
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+    }
+
+    .k-pivotgrid-configurator-header-text {
+        word-break: normal;
+        overflow-wrap: anywhere;
+    }
+
+    .k-pivotgrid-configurator-content {
+        margin: $pivotgrid-configurator-content-padding-y (-$pivotgrid-configurator-content-padding-x);
+        padding: $pivotgrid-configurator-content-padding-y $pivotgrid-configurator-content-padding-x;
+        overflow: auto;
+        flex: 1 1 auto;
+
+        .k-form {
+            .k-label {
+                align-items: flex-start;
+                font-size: 16px;
+                font-weight: 500;
+            }
+            .k-dropdown {
+                max-width: 90px;
+            }
+            .k-textbox {
+                flex-basis: 50%;
+            }
+        }
+
+        // Rows, Cols and Filters
+        .k-row-fields,
+        .k-column-fields,
+        .k-filter-fields {
+            margin-top: map-get($spacing, 2);
+            flex-wrap: wrap;
+
+            > * {
+                margin-bottom: map-get($spacing, 2);
+            }
+
+            > *:last-child {
+                margin-bottom: 0;
+            }
+        }
+
+        // Values
+        .k-value-fields {
+            margin: ($pivotgrid-spacer / 2) 0;
+            display: flex;
+            flex-direction: column;
+            gap: ($pivotgrid-spacer / 2);
+        }
+
+        .k-select-all .k-checkbox {
+            margin-inline-end: map-get($spacing, 1);
+        }
+
+        .k-fields-list-wrapper {
+            flex: 1 1 auto;
+            border-style: solid;
+            border-width: 1px;
+            max-height: 200px;
+            overflow-x: hidden;
+            overflow-y: auto;
+        }
+
+        .k-settings-description {
+            text-align: center;
+            margin-top: 1rem;
+        }
+
+        .k-chip-content {
+            padding-inline-end: map-get($spacing, 1);
+        }
+
+        .k-treeview {
+            padding: 0 map-get($spacing, 2);
+            overflow: auto;
+        }
+    }
+
+    .k-pivotgrid-configurator-actions {
+        margin: (-$pivotgrid-configurator-actions-padding-y) (-$pivotgrid-configurator-actions-padding-x);
```

Comment:
we can remove this negative margin and instead utilize the `k-actions k-hstack k-justify-content-end` classes for the container. See https://github.com/telerik/kendo-themes/commit/9ff8244e9d17b6b473990b00d680548d67248f82 for reference.  

This will also make the spacing in the container consistent, as the currently, the actions wrapper have padding and receive additional padding from the parent container.

Reply:
Thank you for noticing this. I changed the action buttons in the column menus, but forget to do the same for configurator itself.